src/main/java/saasquatch/extintegration/EIHeaders.java

```diff
@@ -0,0 +1,30 @@
+package saasquatch.extintegration;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import javax.annotation.Nonnull;
+import com.google.common.net.HttpHeaders;
+
+public class EIHeaders {
+
+  public static Map<String, String> getGlobalHeaders(@Nonnull Collection<String> frameSrc) {
+    final List<String> csp = new ArrayList<>();
+    csp.add("default-src https: 'unafe-inline' 'unsafe-eval'");
+    if (!frameSrc.isEmpty()) {
+      csp.add("frame-src " + String.join(" ", frameSrc));
```

Comment:
I believe external integrations need to use `frame-ancestors`? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors